src/System.Security.Cryptography.Algorithms/ref/System.Security.Cryptography.Algorithms.csproj

```diff
@@ -2,10 +2,10 @@
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.1.0.0</AssemblyVersion>
+    <AssemblyVersion>4.2.0.0</AssemblyVersion>
     <OutputType>Library</OutputType>
-    <PackageTargetFramework>netstandard1.4</PackageTargetFramework>
-    <NuGetTargetMoniker>.NETStandard,Version=v1.4</NuGetTargetMoniker>
+    <PackageTargetFramework>netstandard1.6</PackageTargetFramework>
```

Comment:
You could call it 1.5 but you'd need to baseline the ApiCompat errors you get for the desktop facade (since these APIs are presumably missing from .NET 4.6.2).  I'm pushing through the buildtools change now.  You should have something ready by tomorrow.